docs/2.7.0/versions.json

```diff
@@ -1,5 +1,5 @@
 {
   "daml": "2.7.0-snapshot.20230228.11494.0.510b7e1f",
   "canton": "20230227",
-  "daml_finance": "1.1.3"
+  "daml_finance": "1.1.4"
```

Comment:
Do you need to change this on the Daml Finance side first? Seems we are currently on 1.1.3 there.

Reply:
i already built `1.1.4` on top of the release docs --> https://github.com/digital-asset/daml-finance/tree/backport-2.6